spec/acceptance/exec_pwsh_spec.rb

```diff
@@ -482,6 +494,27 @@ def platform_string(host, windows, posix)
         its(:content) { should match /puppet/ }
       end
     end
+
+    context 'on POSIX platforms', :if => posix6_agents.count > 0 do
+      p2 = <<-MANIFEST
+      file{'/external-script.ps1':
+        content => '#{File.open(File.join(File.dirname(__FILE__), 'files/get-command-posix.ps1')).read()}'
+      }
+      exec{"TestPowershellPS1":
+        command   => '/external-script.ps1',
+        #{ps_environment}
+        provider  => powershell,
+        require   => File['/external-script.ps1']
+      }
+      MANIFEST
+
+      describe file('/tmp/commands.csv') do
```

Comment:
Is there a Beaker helper we can use for a cross-platform TEMP?

Some mods here, would allow us to again share the tests cross platform.

Reply:
With my new changes this is now redundant.